server/src/test/java/org/elasticsearch/cluster/routing/allocation/DiskThresholdMonitorTests.java

```diff
@@ -128,10 +128,26 @@ protected void updateIndicesReadOnly(Set<String> indicesToMarkReadOnly, ActionLi
             }
         };
 
+        final long totalBytes = testMaxHeadroom ? ByteSizeValue.ofGb(10000).getBytes() : 100;
```

Comment:
When testing the headroom settings we use very large data sizes in contrast to the existing tests that use tiny data sizes instead. Could we keep using small numbers for both, and ideally the same numbers wherever possible?

Reply:
Hi @DaveCTurner , hmmm, I see the point. However, I feel it is better to test the max headroom values for the larger sizes, since the default intention/behavior is that they apply there. This allows to test the default headroom values as well. If I used 100 bytes as total disk, I would need to furthermore specifically configure the max headroom settings to non-default values. Note that there are other tests that test specific headroom values. What do you think? cc @henningandersen as well.

Reply:
I think this comment also arose from my misunderstanding about the effects of these settings earlier. I see now why it makes more sense to leave the settings at their default values, and therefore we need very large disks in these tests.